src/ralph/export_to_ng/management/commands/export_model.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+from optparse import make_option
+
+from django.core.management.base import BaseCommand
+from django.db.models import get_models
+from import_export import resources
+
+import ralph_assets
+from ralph.discovery import models_device
+from ralph.export_to_ng import resources as ralph_resources
+
+SIMPLE_MODELS = [
+    'AssetLastHostname',
+    'Environment',
+    'Service',
+    'Manufacturer',
+    'Category',
+    'ComponentModel',
+    'Warehouse',
+    'DataCenter',
+    'Accessory',
+    # 'CloudProject',
+    'LicenceType',
+    'SoftwareCategory',
+    'SupportType',
+    # Pawel:Database
+    # brak:VIP
+    # brak:VirtualServer
+]
+DEPENDENT_MODELS = [
+    'AssetModel',
+    'GenericComponent',
+    'BackOfficeAsset',
+    'ServerRoom',
+    'Rack',
+    'DataCenterAsset',
+    'Connection',
+    'DiskShare',
+    'DiskShareMount',
+    'Licence',
+    'Support',
+]
+MANY_TO_MANY = [
+    'ServiceEnvironment',
+    'LicenceAsset',
+    'LicenceUser',
+    'RackAccessory',
+]
+
+ALLOWED_MODELS_MSG = """
+Possible models to export are:
+
+Simple models:
+{}
+
+Dependent models:
+{}
+
+ManyToMany Models:
+{}
+""".format(
+    os.linesep.join(SIMPLE_MODELS),
+    os.linesep.join(DEPENDENT_MODELS),
+    os.linesep.join(MANY_TO_MANY),
+)
+
+APP_MODELS = {model._meta.object_name: model for model in get_models()}
+APP_MODELS.update({
+    # exceptions for ambigious models like Warehouse, which is in Scrooge and in
+    # Assets, we need only asset's one so code below:
+    'Warehouse': ralph_assets.models.Warehouse,
+    'Service': models_device.ServiceCatalog,
+})
+
+
+def get_resource(model_name):
+    """Return resource for import model."""
+    resource_name = model_name + 'Resource'
+    resource = getattr(ralph_resources, resource_name, None)
+    if not resource:
+        model_class = APP_MODELS[model_name]
+        print(model_class)
+        resource = resources.modelresource_factory(model=model_class)
+    print(resource)
+    return resource()
+
+
+class Command(BaseCommand):
+    help = os.linesep.join([
+        'Export data in Ralph-NG format.',
+        ALLOWED_MODELS_MSG,
+    ])
+
+    # TODO:: zipped as arg
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '--dst-dir',
```

Comment:
afaik if param has only `--` version it is not required - shouldn't it be required?